src/models/JobRoleResponse.ts

```diff
@@ -0,0 +1,8 @@
+export type JobRoleResponse = {
+    jobRoleId: number,
+    roleName: string,
+    location: string, //enum as string? String or string?
```

Comment:
Would be better to create an enum in models that stores our set of locations and have this reference that type :) 

[](https://www.digitalocean.com/community/tutorials/how-to-use-enums-in-typescript)

Reply:
https://www.digitalocean.com/community/tutorials/how-to-use-enums-in-typescript

Reply:
Will make it easier down the line when we add functionality to register new roles. Can use the enum to populate a consistent dropdown list of potential job locations.